backbone/src/main/java/org/researchstack/backbone/answerformat/EmailAnswerFormat.java

```diff
@@ -1,15 +1,19 @@
 package org.researchstack.backbone.answerformat;
 
 
+import android.text.InputType;
+
 import org.researchstack.backbone.utils.TextUtils;
 
 public class EmailAnswerFormat extends TextAnswerFormat
 {
     private static final int MAX_EMAIL_LENGTH = 255;
 
+    /* Default constructor needed for serilization/deserialization of object */
     public EmailAnswerFormat()
     {
         super(MAX_EMAIL_LENGTH);
+        setInputType(InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS);
```

Comment:
Thought it was odd, but you can't just have InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS without also having InputType.TYPE_CLASS_TEXT.   Same goes for any of the variations of TYPE_NUMBER